### DISCUSSION TITLE
Can this layout be achieved with simple calendar?

### DISCUSSION THREAD

**Discussion by cushtycode** [excid3/simple_calendar#316] [Q&A]
Hi, can a weekly view be achieved with Simple Calendar, just showing the week on the UI, not the other weeks? please see attached image of what we want the UI to look like, just seeing if it is possible, thanks.

<img width="583" alt="Screen Shot 2023-05-26 at 10 05 48" src="https://github.com/excid3/simple_calendar/assets/80784275/8c04f3b3-8f06-47ba-a054-d6da1f187195">

**Comment by excid3** [comment]
Use the Week Calendar that's mentioned in the readme.
